xscen/aggregate.py

```diff
@@ -309,10 +312,14 @@ def compute_deltas(
         deltas[v_name].attrs["delta_reference"] = reference_horizon
 
         for a in ["description", "long_name"]:
-            if hasattr(other_hz[vv], a):
-                deltas[v_name].attrs[
-                    a
-                ] = f"{other_hz[vv].attrs[a]}: {_kind} delta compared to {reference_horizon}."
+            update_attr(
+                deltas[v_name],
+                a,
+                _("{attr1}: {kind} delta compared to {refhoriz}."),
+                others=[other_hz[vv]],
+                refhoriz=reference_horizon,
+                kind=_kind,
```

Comment:
I would change `_kind` above to language-neutral abbreviations: 'abs.', 'rel.', 'pct.'